examples/voyage-ai/embedding.ts

```diff
@@ -0,0 +1,17 @@
+import { VoyageAIEmbedding } from "llamaindex";
```

Comment:
@bjlange to test this, you also need to add @llamaindex/voyage-ai to the package.json of examples